employee-rostering-frontend/src/ui/pages/rotation/EditShiftTemplateModal.tsx

```diff
@@ -0,0 +1,315 @@
+              min={1}
+              max={this.props.rotationLength? this.props.rotationLength : undefined}
+              onChange={(v) => {
+                this.setState(old => ({
+                  editedValue: {
+                    ...old.editedValue,
+                    startDayOffset: v? parseInt(v) - 1 : undefined
+                  }
+                }));
+              }}
+            />
+            <Label>Start Time</Label>
+            <TextInput
+              aria-label="Start Time"
+              type="time"
+              defaultValue={this.state.editedValue.startTime?
+                moment("2018-01-01T00:00").add(this.state.editedValue.startTime.hours, "hours")
+                  .add(this.state.editedValue.startTime.minutes, "minutes").format("HH:mm") 
+                : 
+                undefined}
+              onChange={(v) => {
+                const parts = v.split(":");
+                this.setState(old => ({
+                  editedValue: {
+                    ...old.editedValue,
+                    startTime: {
+                      hours: parseInt(parts[0]),
+                      minutes: parseInt(parts[1])
+                    }
+                  }
+                }));
+              }}
+            />
+          </InputGroup>
+          <InputGroup>
+            <Label>End Day Offset</Label>
+            <TextInput
+              aria-label="End Day Offset"
+              type="number"
+              defaultValue={(this.state.editedValue.endDayOffset !== undefined)? String(
+                this.state.editedValue.endDayOffset + 1) : undefined}
+              min={1}
+              max={this.props.rotationLength? this.props.rotationLength : undefined}
+              onChange={(v) => {
+                this.setState(old => ({
+                  editedValue: {
+                    ...old.editedValue,
+                    endDayOffset: v? parseInt(v) - 1 : undefined
+                  }
+                }));
+              }}
+            />
+            <Label>End Time</Label>
+            <TextInput
+              aria-label="End Time"
+              defaultValue={this.state.editedValue.endTime? 
+                moment("2018-01-01T00:00").add(this.state.editedValue.endTime.hours, "hours")
+                  .add(this.state.editedValue.endTime.minutes, "minutes").format("HH:mm")
+                : 
+                undefined}
+              type="time"
+              onChange={(v) => {
+                const parts = v.split(":");
+                this.setState(old => ({
+                  editedValue: {
+                    ...old.editedValue,
+                    endTime: {
+                      hours: parseInt(parts[0]),
+                      minutes: parseInt(parts[1])
+                    }
+                  }
+                }));
+              }}
+            />
+          </InputGroup>
+          <InputGroup>
+            <Label>Rotation Employee</Label>
+            <TypeaheadSelectInput
```

Comment:
The dropdown associated with this input is rendered inside the dialog whereas it should be on top of it. I think the dropdown's `z-index` just needs to be higher that the modal's one. Maybe it can be fixed upstream (in PatternFly).

Can you fix it in this PR or file a ticket for it?

Reply:
https://issues.jboss.org/browse/PLANNER-1603